### ISSUE TITLE
Module not found: Can't resolve 'algolia-places-react' in '.............../src/components'

### ISSUE THREAD

**Issue by kikiklang** [kontrollanten/algolia-places-react#93]
after installation with  : npm install algolia-places-react --save

i keep getting this error. I'm not sure what's wrong with my component.

```
import React from 'react';
import AlgoliaPlaces from 'algolia-places-react';

    const Places = () => {
      return (
        <AlgoliaPlaces
          placeholder='Write an address here'

          options={ {
            appId: 'my-app-id',
            apiKey: 'sharing-is-caring',
            language: 'sv',
            countries: [ 'se' ],
            type: 'city',
            // Other options from https://community.algolia.com/places/documentation.html#options
          } }

          onChange={ ({ query, rawAnswer, suggestion, suggestionIndex }) => 
            console.log('Fired when suggestion selected in the dropdown or hint was validated.') }

          onSuggestions={ ({ rawAnswer, query, suggestions }) => 
            console.log('Fired when dropdown receives suggestions. You will receive the array of suggestions that are displayed.') }

          onCursorChanged={ ({ rawAnswer, query, suggestion, suggestonIndex }) => 
            console.log('Fired when arrows keys are used to navigate suggestions.') }

          onClear={ () => 
            console.log('Fired when the input is cleared.') }

          onLimit={ ({ message }) => 
            console.log('Fired when you reached your current rate limit.') }

          onError={ ({ message }) => 
            console.log('Fired when we could not make the request to Algolia Places servers for any reason but reaching your rate limit.') }
        />
      );  
    }

    export default Places
```

**Comment by kikiklang** [comment]
Everything has been restarted and  the message i get is : 

```
Failed to compile.

./src/components/places.js
Module not found: Can't resolve 'algolia-places-react' in '/home/vincent/Documents/Web/react-crud/client/src/components'
```
The folder algolia-places-react is  in the node-modules folder.

my package.json looks like this : 

```
{
  "name": "react-crud",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@babel/core": "7.1.6",
    "@svgr/webpack": "2.4.1",
    "algolia-places-react": "^1.3.2",
    "axios": "^0.18.0",
    "babel-core": "7.0.0-bridge.0",
    "babel-jest": "23.6.0",
    "babel-plugin-named-asset-import": "^0.3.0",
    "babel-preset-react-app": "^7.0.0",
    "bfj": "6.1.1",
    "case-sensitive-paths-webpack-plugin": "2.1.2",
    "chalk": "2.4.1",
    "css-loader": "1.0.0",
    "dotenv": "6.0.0",
    "dotenv-expand": "4.2.0",
    "eslint-config-react-app": "^3.0.6",
    "eslint-plugin-flowtype": "2.50.1",
    "eslint-plugin-import": "2.14.0",
    "eslint-plugin-jsx-a11y": "6.1.2",
    "file-loader": "2.0.0",
    "fork-ts-checker-webpack-plugin-alt": "0.4.14",
    "fs-extra": "7.0.0",
    "html-webpack-plugin": "4.0.0-alpha.2",
    "identity-obj-proxy": "3.0.0",
    "jest": "23.6.0",
    "jest-pnp-resolver": "1.0.1",
    "jest-resolve": "23.6.0",
    "leaflet": "^1.4.0",
    "mini-css-extract-plugin": "0.4.3",
    "optimize-css-assets-webpack-plugin": "5.0.1",
    "pnp-webpack-plugin": "1.1.0",
    "postcss-flexbugs-fixes": "4.1.0",
    "postcss-loader": "3.0.0",
    "postcss-preset-env": "6.3.1",
    "postcss-safe-parser": "4.0.1",
    "react": "^16.7.0",
    "react-algolia-places": "^0.1.2",
    "react-app-polyfill": "^0.2.0",
    "react-dev-utils": "^7.0.1",
    "react-dom": "^16.7.0",
    "react-router-dom": "^4.3.1",
    "resolve": "1.8.1",
    "sass-loader": "7.1.0",
    "style-loader": "0.23.0",
    "styled-components": "^4.1.3",
    "styled-reset": "^1.6.6",
    "terser-webpack-plugin": "1.1.0",
    "url-loader": "1.1.1",
    "webpack": "4.19.1",
    "webpack-dev-server": "3.1.14",
    "webpack-manifest-plugin": "2.0.4",
    "workbox-webpack-plugin": "3.6.3"
  },
  "scripts": {
    "start": "node scripts/start.js",
    "build": "node scripts/build.js",
    "test": "node scripts/test.js"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": [
    ">0.2%",
    "not dead",
    "not ie <= 11",
    "not op_mini all"
  ],
  "devDependencies": {
    "babel-eslint": "^9.0.0",
    "babel-loader": "^8.0.4",
    "eslint": "^5.6.0",
    "eslint-loader": "^2.1.1",
    "eslint-plugin-react": "^7.11.1"
  },
  "jest": {
    "collectCoverageFrom": [
      "src/**/*.{js,jsx,ts,tsx}",
      "!src/**/*.d.ts"
    ],
    "resolver": "jest-pnp-resolver",
    "setupFiles": [
      "react-app-polyfill/jsdom"
    ],
    "testMatch": [
      "<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
      "<rootDir>/src/**/?(*.)(spec|test).{js,jsx,ts,tsx}"
    ],
    "testEnvironment": "jsdom",
    "testURL": "http://localhost",
    "transform": {
      "^.+\\.(js|jsx|ts|tsx)$": "<rootDir>/node_modules/babel-jest",
      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
      "^(?!.*\\.(js|jsx|ts|tsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
    },
    "transformIgnorePatterns": [
      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|ts|tsx)$",
      "^.+\\.module\\.(css|sass|scss)$"
    ],
    "moduleNameMapper": {
      "^react-native$": "react-native-web",
      "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy"
    },
    "moduleFileExtensions": [
      "web.js",
      "js",
      "web.ts",
      "ts",
      "web.tsx",
      "tsx",
      "json",
      "web.jsx",
      "jsx",
      "node"
    ]
  },
  "babel": {
    "presets": [
      "react-app"
    ]
  }
}
```

**Comment by kontrollanten** [comment]
Strange, it sounds like webpack related. I saw that you also have `react-algolia-places`, does that one work? Could you publish your repo so I can investigate further? Or if you can reproduce it in another repo.

**Comment by kikiklang** [comment]
React-algolia-places work. I removed it in the meantime

here is the repo : https://github.com/kikiklang/algolia-test.

thx for your help.

**Comment by kontrollanten** [comment]
Okay, I found the problem. Thanks for reporting! It's a bug. You have to import by using `import AlgoliaPlacesReact from 'algolia-places-react/dist'`

**Comment by kontrollanten** [comment]
This should be solved in 1.3.3
